File: Exercise/Easy/13. Square of a Number/Code/Square of a Number.cs
using System;

public class SqrtNumber {
    public static void sqrtNumber(int num) {
        num = num * num;
        Console.WriteLine(num);
    }

    public static void Main(string[] args) {
        int num = int.Parse(Console.ReadLine());

        sqrtNumber(num);
    }
}

File: Exercise/Easy/13. Square of a Number/Code/Square of a Number.java
import java.util.Scanner;

public class SqrtNumber {
    public static void sqrtNumber(int num) {
        System.out.println(num * num);
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int num = scanner.nextInt();
        scanner.close();

        sqrtNumber(num);
    }
}


